migrations/README.rst

```diff
@@ -227,6 +255,27 @@ If you face this error, it means that the :code:`pepys-import` repository should
 
 ------------
 
+.. code-block:: bash
+
+    Database tables are not found! (Hint: Did you initialise the DataStore?)
+
+If you migrated your **SQLite** database and started to see this message, it would have been because of the extra tables created by Alembic. When a table with Geometry column is included in migration,
+please check the SQLite database if there are any temporary tables (with *tmp* keyword) such as :code:`idx_Geometries_tmp_geometry_node` etc. If there is any, please drop these tables. After that,
+please open your migration script and add :code:`spatial_index=False` argument to the Geometry column. An example is as follows:
+
+.. code-block:: python
+
+    # Change this column
+    geometry = deferred(Column(Geometry(geometry_type="GEOMETRY", management=True), nullable=False))
+    # Add spatial_index
```

Comment:
Hang on, this has confused me a bit more - we're saying that we're adding a spatial index here, but we're setting `spatial_index=False`?

Reply:
It's a typo, sorry about that. I meant to add `spatial_index=False` parameter.

Reply:
Fixed:
https://github.com/debrief/pepys-import/pull/427/commits/a68109ce663f06ef89f03955c8773059cb479741